components/FooterSliceZone.vue

```diff
@@ -24,7 +24,11 @@
               :field="socialMedia.socialmedia_url"
               class="footer-social-media__icon"
             >
-              <pix-image :field="socialMedia.socialmedia_image" />
+              <pix-image
+                :field="socialMedia.socialmedia_image"
+                width="30"
+                height="30"
```

Comment:
**problème :**  Nos standards actuels privilégient l'ajout de style dans les classes plutôt qu'en inline
**question/suggestion :** Est-ce que l'ajout des dimensions dans la classe `footer-social-media__icon`  pose problème ?

Reply:
Merci pour la review :)

La [bonne pratique actuelle](https://web.dev/optimize-cls/#modern-best-practice) est de préciser les attributs `width` et `height` des `img` pour que le navigateur réserve au plus tôt l'espace qui sera nécessaire à l'image une fois chargée. On évite ainsi le *layout shift* que l'on peut voir dans [ces exemples](https://web.dev/optimize-cls/#images-without-dimensions) 😄 

Pour info la mesure de performance de Lighthouse alerte sur cette pratique.

Reply:
Intéressant, je ne connaissais pas. J'y lis aussi que : 

> If your image is in a container, you can use CSS to resize the image to the width of this container. We set height: auto; to avoid the image height being a fixed value (for example 360px).

Ça semble correspondre à nos pratiques actuelles tout en suivant le standard moderne et en répondant aux problèmes de Lighthouse d'après eux (je n'ai pas vérifié)

La question qui découle de ça : est-ce qu'on souhaite privilégier une approche par container (ici le `pix-link` voire `pix-image` d'ailleurs) ou par "instance d'image" ? 

À voir ce qu'en pense l'équipe